pkg/providers/aws/provider_blobstorage.go

```diff
@@ -465,14 +465,27 @@ func getS3buckets(s3svc s3iface.S3API) ([]*s3.Bucket, error) {
 	return existingBuckets, nil
 }
 
-func reconcileS3BucketSettings(bucket string, s3svc s3iface.S3API) error {
+func reconcileS3BucketSettings(bucket string, s3svc s3iface.S3API, bs *v1alpha1.BlobStorage) error {
+
+	var blockPublicAcls = defaultBlockPublicAcls
+	var blockPublicPolicy = defaultBlockPublicPolicy
+	var ignorePublicAcls = defaultIgnorePublicAcls
+	var restrictPublicBuckets = defaultRestrictPublicBuckets
+
+	if (bs.Spec.RemoveBlockOnPublicAccess) {
+		blockPublicAcls = false
+		blockPublicPolicy = false
+		ignorePublicAcls = false
+		restrictPublicBuckets = false
+	}
+
 	_, err := s3svc.PutPublicAccessBlock(&s3.PutPublicAccessBlockInput{
 		Bucket: aws.String(bucket),
 		PublicAccessBlockConfiguration: &s3.PublicAccessBlockConfiguration{
-			BlockPublicAcls:       aws.Bool(defaultBlockPublicAcls),
```

Comment:
These defaults were an ask from BU not to have anything public, I know this is a customer issue, but has it been highlighted to the BU that this change is happening.

Reply:
Waiting for feedback from BU, thanks.